docs/introduction/2-getting-started.mdx

```diff
@@ -16,22 +16,23 @@ It will get generate a new [Next.js](https://nextjs.org/) app with Circuit UI an
 
 ## Adding Circuit UI to an existing app
 
-Already have a project? You can install Circuit UI via `yarn` or `npm`. In your current project directory, run the following command:
+Already have a project? You can install Circuit UI via `npm` or `yarn`. In your current project directory, run the following command:
```

Comment:
Should we still recommend yarn at all? This might just work on yarn v1 but it's kind of unmaintained 👀

Reply:
(I think I also raised this in foundry last week and you probably addressed it there, I haven't checked yet and will do after I send this review 😛)

Reply:
☝️ yup, ref: https://github.com/sumup-oss/foundry/pull/834#discussion_r1154545936